signedxml_test.go

```diff
@@ -225,7 +225,7 @@ func TestEnvelopedSignatureProcess(t *testing.T) {
 	})
 
 	Convey("Given an unparented signature element", t, func() {
-		doc := "<Signatrue></Signature>"
+		doc := "<Signature></Signature>"
```

Comment:
Funny how this typo is from 2016 https://github.com/moov-io/signedxml/commit/ca36cbd87bad62354b7182bb3d32c9c9101008aa#

Reply:
Agree. Not sure why this started causing tests to fail.